src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/timeline/TimelineEventEntity.kt

```diff
@@ -107,4 +107,7 @@ enum class TimelineEventType {
   STEP_NOT_STARTED,
   STEP_STARTED,
   STEP_COMPLETED,
+  PRISON_ADMISSION,
+  PRISON_RELEASE,
+  PRISON_TRANSFER,
```

Comment:
I've added these to the entity version of the enum, even though they won't be saved for the foreseeable future. It made the `MapStruct` mapping considerably easier, as you can't simply use `ignore = true` within `@ValueMapping`.